modules/videopress/class.videopress-player.php

```diff
@@ -326,10 +326,14 @@ private function html5_static() {
 				$html .= '<source src="' . esc_url( $mp4 ) . '" type="video/mp4; codecs=&quot;' . esc_attr( $this->video->videos->mp4->codecs ) . '&quot;" />';
 			unset( $mp4 );
 		}
-		$ogg = $this->video->videos->ogv->url;
-		if ( ! empty( $ogg ) )
-			$html .= '<source src="' . esc_url( $ogg ) . '" type="video/ogg; codecs=&quot;' . esc_attr( $this->video->videos->ogv->codecs ) . '&quot;" />';
-		unset( $ogg );
+
+		if ( isset( $this->video->videos->ogv ) ) {
+			$ogg = $this->video->videos->ogv->url;
+			if ( ! empty( $ogg ) ) {
+				$html .= '<source src="' . esc_url($ogg) . '" type="video/ogg; codecs=&quot;' . esc_attr($this->video->videos->ogv->codecs) . '&quot;" />';
```

Comment:
This is all previously written code. If you look at the original version, the whole file is improperly spaced. Are you asking me to respace the whole file?

The only lines I actually added were: `if ( isset( $this->video->videos->ogv ) ) {` and `}`